packages/schema-generator/API.md

```diff
@@ -12,6 +12,7 @@ export class SchemaGenerator {
 
 // @public (undocumented)
 export type SchemaGeneratorConfig = {
+    secretName: string;

```

Comment:
The API as is says that both are mandatory parameters.
If I understand this correctly the customer interacting with CLI won't get to this level of details and they'll just operate at "backend secret" abstraction level as explained by @sundersc .

At the generator abstraction level however it seems that both parameters are related and mandatory (i.e. resolved by cli and passed down). 

Therefore:
Can we call it `connectionUriSecretName` ?
Or if both are required and related can we group them together in a form of:
```
export type SchemaGeneratorConfig = {
  connectionUri: {
     secretName: string;
     value: string;
  }
}
```

It would be best if relationship between these two were visible while reading this API. Additionally if there's area that can't be easily expressed by API (the upstream data flow for instance) it should be explained in jsdocs for this type.

Reply:
Combined the secret name and the value into a single property `connectionUri` as suggested.